internal/ingress/annotations/mirror/main.go

```diff
@@ -85,5 +90,18 @@ func (a mirror) Parse(ing *networking.Ingress) (interface{}, error) {
 		config.Source = ""
 	}
 
+	config.Host, err = parser.GetStringAnnotation("mirror-host", ing)
+	if err != nil  {
+		if config.Target != "" {
+			url, err := parser.StringToURL(config.Target)
+			if err != nil {
+				config.Host = ""
+			}
```

Comment:
Need an else here, otherwise you will move forward on your logics and get a panic, as url is gonna be a null structure due to previous error

Reply:
Thank you guys - my fool mistake (
I have fixed it and have added some tests